2021-02-01-infinispan-cli-image.adoc

```diff
@@ -0,0 +1,49 @@
+---
+layout: blog
+title: Infinispan Native CLI
+permalink: /blog/:year/:month/:day/infinispan-12-final
+date: '2021-02-01T12:00:00.000-00:00'
+author: remerson
+tags: [ "docker", "cli", "containers", "quarkus", "native"]
+---
+
+Starting with Infinispan 12, we now provide a natively compiled version of the Infinispan CLI.
+
+This can be consumed as a native binary or via a container image.
+
+== Usage
+. Start a Infinispan server instance so that the CLI has an endpoint to connect to:
+----
+docker run --net=host -it -p 11222:11222 -e USER="user" -e PASS="pass" quay.io/infinispan/server:12.0
```

Comment:
```suggestion
docker run --net=host -it --rm -p 11222:11222 -e USER="user" -e PASS="pass" quay.io/infinispan/server:12.0
```

Reply:
`--net=host` isn't required for the server. `-p` forwards the port 11222 from the host machine to the container (at least on my machine)

Reply:
Good catch!